docs/README.md

```diff
@@ -257,3 +257,189 @@ To disable that behavior, use the `compactMode` parameter in JSON-RPC request:
+            "sourceId": "wasapi_output_capture_30c2879c",
+            "muted": false,
+            "name": "DesktopAudioDevice1",
+            "type": "wasapi_output_capture",
+            "id": "wasapi_output_capture_30c2879c-989e-4230-a293-c2e47941197a"
+        },
+       {
+            "_type": "HELPER",
+            "resourceId": "AudioSource[\"wasapi_input_capture_d497319b\"]",
+            "sourceId": "wasapi_input_capture_d497319b",
+            "muted": false,
+            "name": "AuxAudioDevice1",
+            "type": "wasapi_input_capture"
+            "id": "wasapi_input_capture_d497319b"
+        }
+    ]
+}
+```
+
+Usually the AudioSource with name `AuxAudioDevice1` is mike.
+To mute it you have to call `IAudioSourceApi.setMuted(true)`.
+Use `resourceId` as the reference to `IAudioSourceApi` instance: 
+
+### Request
+```
+{
+    "jsonrpc": "2.0",
+    "id": 7,
+    "method": "setMuted",
+    "params": {
+        "resource": "AudioSource[\"wasapi_input_capture_d497319b\"]",
+        args: [true]
+    }
+}
+```
+
+## How to hide/show a scene item?
+
+First get the active scene
+### Request
+```
+{
+    "jsonrpc": "2.0",
+    "id": 8,
+    "method": "activeScene",
+    "params": {
+        "resource": "ScenesService"
+    }
+}
+```
+
+### Response
+```
+{
+    "jsonrpc": "2.0",
+    "id": 8,
+    "result": {
+        "_type": "HELPER",
+        "resourceId": "Scene[\"scene_02da7db7\"]",
+        "id": "scene_02da7db7",
+        "name": "Scene",
+        "items": [
+           {
+                "sceneItemId": "b18ee2b0",
+                "sourceId": "color_source_61221191",
+                "x": 975.2116452561818,
+                "y": 392.0672268907563,
+                "visible": true
+            },
+           {
+                "sceneItemId": "fffadcdd",
+                "sourceId": "image_source_21071a85",
+                "x": 331.0344827586205,
+                "y": 174.25210084033603,
+                "visible": true
+            }
+        ]
+    }
+}
+```
+
+Lets hide the first scene item via `ISceneItemApi.setVisibility(false)` method.
+To do that you need a `resourceId` for this scene item.
+Use `ISceneApi.getSceneItem(sceneItemId)`.
+
+### Request
+```
+{
+    "jsonrpc": "2.0",
+    "id": 9,
+    "method": "getItem",
+    "params": {
+        "resource": "Scene[\"scene_02da7db7\"]",
+        "args": ["b18ee2b0"]
+    }
+}
+```
+
+### Response
+```
+{
+    "jsonrpc": "2.0",
+    "id": 9,
+    "result": {
+        "_type": "HELPER",
+        "resourceId": "SceneItem[\"scene_02da7db7\",\"b18ee2b0-6ff7\",\"color_source_61221191\"]",
+        "sceneItemId": "b18ee2b0",
+        "sourceId": "color_source_61221191",
+        "x": 975.2116452561818,
+        "y": 392.0672268907563,
+        "scaleX": 1,
+        "scaleY": 1,
+        "visible": true
+    }
+}
+```
+
+Now you have a `resourceId` for `ISceneItemApi` instance and can use it's methods includes `setVisibility`.
```

Comment:
Now you have a `resourceId` for the `ISceneItemApi` instance and can use its methods including `setVisibility`.